app/profile/components/payment-methods-tab.tsx

```diff
@@ -79,8 +79,7 @@ export default function PaymentMethodsTab() {
 
       try {
         data = JSON.parse(responseText)
-      } catch (error) {
-        console.log(error);
+      } catch (e) {
```

Comment:
this will throw an eslint error. Please make sure that `e` is being handled here.